client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/style.scss

```
@@ -23,6 +26,21 @@
 	inset-block-start: 50px;
 }
 
+.notification-settings__bulk-edit-warning {
+	border-left: 4px solid var(--alert-yellow);
+	padding: 12px;
+	padding-left: 0;
+	background-color: var(--alert-yellow-light);
+	.notification-settings__bulk-edit-warning-block {
+		.notification-settings__bulk-edit-warning-content {
+			padding-left: 12px;
+			font-weight: 400;
+			font-size: 0.75rem;
+			line-height: 24px;
+		}
+	}
+}
+
```

If we remove the nested div, we can write the style as below

```suggestion
.notification-settings__bulk-edit-warning {
	border-left: 4px solid var(--alert-yellow);
	padding: 12px;
	background-color: var(--alert-yellow-light);
	font-weight: 400;
	font-size: 0.75rem;

}
```